frontend/public/components/operator-lifecycle-manager/operator-group.tsx

```diff
@@ -30,7 +30,9 @@ export const OperatorGroupSelector: React.SFC<OperatorGroupSelectorProps> = (pro
   desc="OperatorGroups"
   placeholder="Select Operator Group"
   selectedKeyKind={referenceForModel(OperatorGroupModel)}
-  resources={[{kind: referenceForModel(OperatorGroupModel)}]} />;
+  resources={props.namespace ?
+    [{kind: referenceForModel(OperatorGroupModel), namespace: props.namespace}]
```

Comment:
You shouldn't need a ternary here, just add `namespace: props.namespace` (will work when `props.namespace` is `undefined`).

Reply:
Good to know! I actually need to make changes to filter out olm-operators instead of only allow global-operators. Any chance this is possible using the Firehose or do I just filter once the call returns?

Reply:
I'm pretty sure you can use a [field selector](https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/#supported-operators) on `metadata.name` for that.